Portfolio/index.html

```diff
@@ -1,13 +1,34 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
+    <title>Resume</title>
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <!-- Write your code below for PORTFOLIO -->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
+  <body>
```

Comment:
The opening tag for the form element should have an action attribute to specify the action URL